src/axom/quest/tests/quest_pro_e_reader.cpp

```diff
@@ -0,0 +1,299 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "axom/core/utilities/FileUtilities.hpp"
+#include "axom/mint/utils/vtk_utils.hpp"  // for write_vtk
+#include "axom/quest/readers/ProEReader.hpp"
+#include "axom/slic.hpp"
+
+// gtest includes
+#include "gtest/gtest.h"
+
+#include <fstream>
+
+//------------------------------------------------------------------------------
+// HELPER METHODS
+//------------------------------------------------------------------------------
+namespace
+{
+/*!
+ * \brief Generates a Pro/E file consisting of a single tetrahedron
+ * \param [in] file the name of the file to generate.
+ * \pre file.empty() == false
+ */
+void generate_pro_e_file(const std::string& file)
+{
+  EXPECT_FALSE(file.empty());
+
+  std::ofstream ofs(file.c_str());
+  EXPECT_TRUE(ofs.is_open());
+
+  ofs << "# Comment header to ignore" << std::endl;
+  ofs << "# Another comment" << std::endl;
+  ofs << "4 1" << std::endl;
+  ofs << "node1 -1.0 0.0 0.0" << std::endl;
+  ofs << "node2 1.0 0.0 0.0" << std::endl;
+  ofs << "node3 0.0 1.0 0.0" << std::endl;
+  ofs << "node4 0.0 0.0 1.0" << std::endl;
+  ofs << "tet1 node1 node2 node3 node4" << std::endl;
```

Comment:
Are you sure the vertex references can be strings? I'd imagine that they should be integers.

Clearly your implementation supports this, but it doesn't seem right to me.

Reply:
> Are you sure the vertex references can be strings? I'd imagine that they should be integers.

Good catch! Yes, they are definitely integers. I'll change that.